app/src/main/java/net/skyscanner/backpack/demo/stories/BarChartStory.kt

```diff
@@ -0,0 +1,61 @@
+package net.skyscanner.backpack.demo.stories
+
+import android.os.Bundle
+import android.view.View
+import net.skyscanner.backpack.barchart.BpkBarChart
+import net.skyscanner.backpack.demo.R
+import java.util.*
+import kotlin.collections.ArrayList
+
+class BarChartStory : Story() {
+
+  override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+    super.onViewCreated(view, savedInstanceState)
+
+    view.findViewById<BpkBarChart>(R.id.bar_chart).apply {
+      invoke(BpkBarChart.Model(
```

Comment:
I think at least for the external api we should avoid the `consumer` approach and call this `setModel`